modules/tracker/models/base/AggregateMetricSpecModelBase.php

```diff
@@ -35,8 +35,10 @@ public function __construct()
             'name' => array('type' => MIDAS_DATA),
             'description' => array('type' => MIDAS_DATA),
             'spec' => array('type' => MIDAS_DATA),
-            'value' => array('type' => MIDAS_DATA),
-            'comparison' => array('type' => MIDAS_DATA),
```

Comment:
This confused me for a bit, but I can see it's vestigial cleanup from the 2.0.1 upgrade. I'm saying that here to confirm my understanding is correct.